src/Form/Formbot.js

```diff
@@ -35,6 +38,7 @@ export default class Formbot extends React.Component {
   static defaultProps = {
     initialValues: {},
     validations: {},
+    validationSchema: {},
```

Comment:
Should this have a `fieldName` property?

Reply:
no, the proptypes aren't doing the right thing so i'll just rm